syntax = "proto3";
package rdma;

message GetAllNodeIDsRequest {
    uint64 node_id = 1;
}

